sources/api/types.go

```diff
@@ -57,19 +57,27 @@ func (a *AggregateData) Merge(b *AggregateData) {
 	a.Events = append(a.Events, b.Events...)
 }
 
+type ContainerSpec struct {
+	cadvisor.ContainerSpec
+	CpuRequest    int64
```

Comment:
Can we have these 2 grouped in a struct?
Would it be good to have CpuLimit as well?